session/bootstrap.go

```diff
@@ -163,6 +163,7 @@ const (
 		Column_name	CHAR(64),
 		Timestamp	TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
 		Column_priv	SET('Select','Insert','Update','References'),
+		KEY             IDX(Host, lower(DB), User, lower(Table_name), lower(Column_name)),
```

Comment:
Can you try to convert existing data to lowercase and use primary key + lowercased data to access the table, rather than add this index? Index is not the best way in this scenario

Reply:
Done please check, THKS